README.md

```diff
@@ -72,9 +72,24 @@ By default, view nodes are strongly typed. The type of a container includes the
 
 The type erasure of View nodes is not an easy trick, as the trait has two associated types and the `rebuild` method takes the previous view as a `&Self` typed parameter. Nonetheless, it is possible. (As far as I know, Olivier Faure was the first to demonstrate this technique, in [Panoramix], but I'm happy to be further enlightened)
 
+## Prerequisites
+
```

Comment:
```suggestion

## Git deps 
```sh
cargo update
# at the moment dependencies like glazier are through git and need to be pulled in manually

```